rer/cookieconsent/profiles/default/registry.xml

```diff
@@ -1,3 +1,26 @@
 <registry>
     <records interface="rer.cookieconsent.controlpanel.interfaces.ICookieConsentSettings" />
+
+    <!-- RESOURCE -->
+    <records prefix="plone.resources/rer.cookieconsent"
+             interface='Products.CMFPlone.interfaces.IResourceRegistry'>
+        <value key="js">++plone++rer.cookieconsent/cookiepolicy.js</value>
+        <value key="css">
+          <element>++plone++rer.cookieconsent/cookiepolicy.css</element>
+        </value>
+    </records>
+
+    <!-- BUNDLE -->
+    <records prefix="plone.bundles/rer.cookieconsent-bundle"
+              interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+      <value key="enabled">True</value>
+      <value key="resources" purge="false">
+        <element>rer.cookieconsent</element>
+      </value>
+      <value key="jscompilation">++plone++rer.cookieconsent/cookiepolicy.js</value>
```

Comment:
It would be awesome to serve a minified version of js and css files with the bundle, but we can also handle this in another issue (and PR), if you prefer